README.md

```diff
@@ -116,7 +116,7 @@ This is a schematic logging configuration to show log messages from input_nameA
 - `relp` type - `relp` input supports receiving logs from the remote logging system over the network using relp.<br>
   **available options**
   - `port`: Port number Relp is listening to. Default to `20514`.
-  - `use_cert`: If true, key/certificates are used. Triplets {`ca_cert`, `cert`, `private_key`} and/or {`ca_cert_src`, `cert_src`, `private_key_src`} should be configured. Default to `true`.
+  - `tls`: If true, key/certificates are used. Triplets {`ca_cert`, `cert`, `private_key`} and/or {`ca_cert_src`, `cert_src`, `private_key_src`} should be configured. Default to `true`.
```

Comment:
"If true, encrypt the connection with TLS.  You must provide key/certificates, and triplets ....."